README.md

```diff
@@ -8,7 +8,19 @@ A Docker Machine driver for the Grid5000 testbed infrastructure. It will provisi
 
 You need a Grid5000 account to use this driver. See [this page](https://www.grid5000.fr/mediawiki/index.php/Grid5000:Get_an_account) to create an account.
 
-## Installation
+## Installation from GitHub releases
+Binary releases are available for Linux, MacOS and Windows on the [releases page](https://github.com/Spirals-Team/docker-machine-driver-g5k/releases).  
+You can use the usual commands to install or upgrade the driver :
+
+```bash
+# download the binary for your OS
+sudo curl -L -o /usr/local/bin/docker-machine-driver-g5k "<link to release>"
```

Comment:
Add a link to https://github.com/Spirals-Team/docker-machine-driver-g5k/releases behind `<link to release>`